lib/scrabble.rb

```diff
@@ -15,4 +27,27 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def prepare(word)
+    if word.class != Array
+      word.upcase!
+      word = word.chars
```

Comment:
No need to set this equal to the variable word. If we get into this branch of the conditional, line 34 would be the last line of code that we run, and `word.chars` would be returned.